tests/smoke/sandbox/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,8 @@
+name: test-snapd-sh
+summary: A no-strings-attached, no-fuss shell for writing tests
+version: 1.0
+
+apps:
+    with-home-plug:
```

Comment:
Bunch of the tests assume that there is an app named just like the snap, so `test-snapd-sh` is all you need to run.
```
2019-11-27 23:30:46 Failed tasks: 47
    - google:ubuntu-14.04-64:tests/main/fakestore-install
    - google:ubuntu-14.04-64:tests/main/interfaces-personal-files
    - google:ubuntu-14.04-64:tests/main/interfaces-system-files
    - google:ubuntu-14.04-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-14.04-64:tests/smoke/sandbox
    - google:ubuntu-16.04-32:tests/main/fakestore-install
    - google:ubuntu-16.04-32:tests/main/interfaces-personal-files
    - google:ubuntu-16.04-32:tests/main/interfaces-system-files
    - google:ubuntu-16.04-32:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-16.04-32:tests/main/system-usernames-illegal
    - google:ubuntu-16.04-32:tests/smoke/sandbox
    - google:ubuntu-16.04-64:tests/main/fakestore-install
    - google:ubuntu-16.04-64:tests/main/interfaces-personal-files
    - google:ubuntu-16.04-64:tests/main/interfaces-system-files
    - google:ubuntu-16.04-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-16.04-64:tests/main/system-usernames-illegal
    - google:ubuntu-16.04-64:tests/smoke/sandbox
    - google:ubuntu-18.04-64:tests/main/fakestore-install
    - google:ubuntu-18.04-64:tests/main/interfaces-personal-files
    - google:ubuntu-18.04-64:tests/main/interfaces-system-files
    - google:ubuntu-18.04-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-18.04-64:tests/main/system-usernames-illegal
    - google:ubuntu-18.04-64:tests/smoke/sandbox
    - google:ubuntu-19.04-64:tests/main/fakestore-install
    - google:ubuntu-19.04-64:tests/main/interfaces-personal-files
    - google:ubuntu-19.04-64:tests/main/interfaces-system-files
    - google:ubuntu-19.04-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-19.04-64:tests/main/system-usernames-illegal
    - google:ubuntu-19.04-64:tests/smoke/sandbox
    - google:ubuntu-19.10-64:tests/main/fakestore-install
    - google:ubuntu-19.10-64:tests/main/interfaces-personal-files
    - google:ubuntu-19.10-64:tests/main/interfaces-system-files
    - google:ubuntu-19.10-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-19.10-64:tests/main/system-usernames-illegal
    - google:ubuntu-19.10-64:tests/smoke/sandbox
    - google:ubuntu-core-16-64:tests/main/fakestore-install
    - google:ubuntu-core-16-64:tests/main/interfaces-personal-files
    - google:ubuntu-core-16-64:tests/main/interfaces-system-files
    - google:ubuntu-core-16-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-core-16-64:tests/main/system-usernames-illegal
    - google:ubuntu-core-16-64:tests/smoke/sandbox
    - google:ubuntu-core-18-64:tests/main/fakestore-install
    - google:ubuntu-core-18-64:tests/main/interfaces-personal-files
    - google:ubuntu-core-18-64:tests/main/interfaces-system-files
    - google:ubuntu-core-18-64:tests/main/parallel-install-common-dirs-undo
    - google:ubuntu-core-18-64:tests/main/system-usernames-illegal
    - google:ubuntu-core-18-64:tests/smoke/sandbox
2019-11-27 23:30:46 Failed task prepare: 8
    - google:ubuntu-14.04-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-16.04-32:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-16.04-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-18.04-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-19.04-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-19.10-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-core-16-64:tests/main/interfaces-snapd-control-with-manage
    - google:ubuntu-core-18-64:tests/main/interfaces-snapd-control-with-manage
error: unsuccessful run
```

in this case the snap meta should probably be:
```
name: test-snapd-sh
summary: A no-strings-attached, no-fuss shell for writing tests
version: 1.0

apps:
    test-snapd-sh:
        command: bin/sh
        plugs: [home]
````

Reply:
@bboozzoo All the issues were fixed, thanks for the review